packages/web-twig/src/Resources/components/Tooltip/stories/TooltipOnHover.twig

```diff
@@ -1,51 +1,53 @@
-<TooltipWrapper>
-    <ButtonLink
-        href="#"
-        aria-describedby="my-tooltip-hover-top"
-        UNSAFE_className="TooltipTarget"
-    >
-        Tooltip on top
-    </ButtonLink>
-    <Tooltip id="my-tooltip-hover-top" placement="top">
-        Hello there!
-    </Tooltip>
-</TooltipWrapper>
+<DocsStack stackAlignment="stretch">
```

Comment:
Same as in react - you should use `stackAligment` on parent `DocsSection`

Reply:
Fixed.